en_us.lang

```diff
@@ -366,21 +366,21 @@ gui.xaero_entity_radar_color=Color:
 gui.xaero_entity_radar_visiblity=Visibility:
 gui.xaero_entity_radar_names=Names:
 gui.xaero_entity_radar_icons=Icons/Heads:
-gui.xaero_box_entity_radar_names=Render entity names under the dots. \n \n Option "Player List" - render while the "List Players" key binding is pressed (TAB by default).
+gui.xaero_box_entity_radar_names=Render entity names under the dots. \n \n Option "Player List" - render only when the "List Players" key binding (TAB by default) is pressed.
```

Comment:
Since version 20.27.0 this tooltip doesn't cover the case where someone has configured the minimap to use mob heads instead of the default dots. What do you think about using something like "Render entity names under the dots or icons/heads"?

Reply:
I think just dots is fine but I don't see why not.

Reply:
I don't need more clarification either, but I suppose that without it, this tooltip may be confusing for some people.

Reply:
Then you can add it, I'm not against it :D

Reply:
Done.